### PR TITLE
Update version of pug

### PR DESCRIPTION
**Pull request by millerds**
Fixing vulnerability warning for pug (other changes in package-lock file were done automatically).

### CODE REVIEW

**Comment by akrantz** [comment]
Just a heads up, in general you should create a topic branch and push that to your fork instead of using the master branch on your fork.
```
git checkout -b branch-name
git push -u fork branch-name
```
where `branch-name` is the name of the branch, and `fork` is the remote for your fork.

**Comment by akrantz** [comment]
Hey, I thought we were just going to remove `pug`.

File: packages/office-addin-sso/package-lock.json [hunk 1]
```diff
@@ -316,24 +323,16 @@
...
 			"bin": {
 				"acorn": "bin/acorn"
 			},
 			"engines": {
 				"node": ">=0.4.0"
 			}
 		},
-		"node_modules/acorn-globals": {
-			"version": "3.1.0",
-			"resolved": "https://office.pkgs.visualstudio.com/_packaging/Office/npm/registry/acorn-globals/-/acorn-globals-3.1.0.tgz",
```
Comment (akrantz) [inline packages/office-addin-sso/package-lock.json:331]: This was the problem where it was pulling from the Office npm registry, which it shouldn't.

**Comment by TCourtneyOwen** [comment]
Mac agent isn't running - just rebooted

**Review by TCourtneyOwen — APPROVED** [review]

**Pull request merged**